client/src/components/shared/search.js

```diff
@@ -22,18 +23,24 @@ class SearchComponent extends Component{
      * CHECKS TARGET VALUE AND SETS PLACEHOLDER AND SEARCH TYPE STATE
      */
     setPlaceholder(event){
+        this.setState({
+            inputValue: []
+        });
         let target = event.target.value;
         if (target === 'members') {
-            this.setState({
-                placeholderText: 'Enter a member name..',
-                searchType: target,
-                searchResults: InputSearchService.fetchSearch(target)
-            })
+            axios.get('http://localhost:3004/member-list')
+                .then(result => {
+                    this.setState({
+                        searchType: target,
+                        placeholderText: 'Enter a member name...',
+                        searchResults: result.data
+                    })
+                })
         } else {
             this.setState({
                 placeholderText: 'Enter a keyword..',
                 searchType: target,
-                searchResults: InputSearchService.fetchSearch(target)
+                searchResults: ['birth control', 'gun control', 'a.c.a']
```

Comment:
No changes need here right now, but eventually on initialization, let's pull all the member names and issues and store them in memory. That way the app doesn't have to make a server call with every key stroke for the auto complete feature.